packages/cardano-graphql/src/WalletProvider/queryTransactions/graphqlTransactionsToCore.ts

```diff
@@ -32,7 +32,7 @@ const scriptToCore = (script: GraphqlScript): Cardano.Script => {
     return { native: nativeScriptToCore(script) };
   } else if (script.__typename === 'PlutusScript') {
     return {
-      plutus: script.cborHex
+      'plutus:v1': script.cborHex
```

Comment:
Should probably do something like:
```
[script.type === 'PlutusScriptV1' ? 'plutus:v1' : 'plutus:v2']: script.cborHex
```

Reply:
yeah I was looking for a field that would give me a classification, but there's no `.type` property on `script` or similar. Do you mean refactor to make this possible?

Reply:
No need to refactor, just add it to [query](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/cardano-graphql/src/operations/transactions.graphql#L102-L105), then `yarn build && yarn generate` and it should be available